Generate.py

```diff
@@ -478,20 +478,18 @@ def roll_settings(weights: dict, plando_options: PlandoOptions = PlandoOptions.b
             handle_option(ret, game_weights, option_key, option, plando_options)
     if PlandoOptions.items in plando_options:
         ret.plando_items = game_weights.get("plando_items", [])
-    if ret.game == "Minecraft" or ret.game == "Ocarina of Time":
-        # bad hardcoded behavior to make this work for now
-        ret.plando_connections = []
-        if PlandoOptions.connections in plando_options:
-            options = game_weights.get("plando_connections", [])
-            for placement in options:
-                if roll_percentage(get_choice("percentage", placement, 100)):
-                    ret.plando_connections.append(PlandoConnection(
-                        get_choice("entrance", placement),
-                        get_choice("exit", placement),
-                        get_choice("direction", placement)
-                    ))
-    elif ret.game == "A Link to the Past":
+    if ret.game == "A Link to the Past":
         roll_alttp_settings(ret, game_weights, plando_options)
+    elif PlandoOptions.connections in plando_options:
```

Comment:
From what I can tell, old ALttP behavior for plando connections is that if you leave off `direction` it defaults to `both`, while other games require direction to be explicit. This may actually be desired behavior for the regular plando connections though, and the only other change within the ALttP section is using `get_choice_legacy`, which would be a safe swap since `on` and `off` are not valid for plando connections (and are caught).

Reply:
Do we want `direction: both` as a default? I could change it to do that and remove the elif, yeah. And yeah this really should be a proper `Option`, I just didn't want to do it at the time.